### ISSUE TITLE
One of the Sub Menu isn't loading

### ISSUE THREAD

**Issue by Walshest** [tlkenergy/sim-tlk#45]
For some reason one of our sub menus isn't loading properly... 
![Bug_withEvaporatorSubMenu](https://github.com/tlkenergy/sim-tlk/assets/45364510/b23dda53-4142-4c48-a8fd-fa6c7af6f301)

**Comment by mateokurti** [comment]
Two of the parameters on this sub-menu have a dependency on `VariableAirInletTemp_int` which doesn't seem to exist as a parameter. Is this intentional? Removing/correcting this dependency should resolve the issue. Meanwhile, we'll work to handle this validation so it is prevented on the FMU Interface level.

**Comment by Walshest** [comment]
Thanks for the fast response.
Actually the parameter does exist... I just downloaded it from Django and searched for it: 
<img width="479" alt="image" src="https://github.com/tlkenergy/sim-tlk/assets/45364510/d4c1c7d4-a507-4841-9a6a-7e32645414c8">
I assume it isn't showing in Sim.TLK anymore as it seems to have changed to a constant in the last compile from Dymola

**Comment by mateokurti** [comment]
Yes, only `parameters` and `inputs` can be configured. That's why it isn't showing on the list.

**Comment by Walshest** [comment]
Thanks for the solution on why it isn't working. 
It would be great if you can leave this bug open until you have implemeneted a warning in the UI to identify this :)
